README.md

```diff
@@ -74,19 +74,20 @@ python run.py --execution-provider coreml
 ```
 
 ### [](https://github.com/s0md3v/roop/wiki/2.-Acceleration#coreml-execution-provider-apple-legacy)CoreML Execution Provider (Apple Legacy)
+Metal support has been added for improved performance on macOS devices.
```

Comment:
wrong section? this is listed under "Apple Legacy" when it should be apple silicon maybe?